Marlin/src/HAL/STM32F1/HAL.h

```diff
@@ -109,6 +109,8 @@
   #else
     #error "LCD_SERIAL_PORT must be -1 or from 1 to 3. Please update your configuration."
   #endif
+
+  #define SERIAL_GET_TX_BUFFER_FREE LCD_SERIAL.availableForWrite
```

Comment:
Is this change necessary in this commmit? It seems unrelated to me.

Reply:
This could go in either commit really but will be beneficial for any serial display on this board so i dropped it here.